src/applications/gi/utils/data/religiousAffiliations.js

```diff
@@ -0,0 +1,70 @@
+export const religiousAffiliations = {
```

Comment:
why not have the object look like
```
export const religiousAffiliations = {
    22: 'American Evangelical Lutheran Church',
    24: 'African Methodist Episcopal Zion Church'
.
.
.
```

then getting the name is a simple `religiousAffiliations[relId]` and when this object is used for the filter we can do `Object.values(religiousAffiliations).sort()`